x/fundraising/keeper/auction.go

```diff
@@ -71,6 +75,9 @@ func (k Keeper) AllocateSellingCoin(ctx sdk.Context, auction types.AuctionI, mIn
 	// Allocate coins to all matched bidders in AllocationMap and
 	// set the amounts in trasnaction inputs and outputs from the selling reserve account
 	for bidder, allocAmt := range mInfo.AllocationMap {
+		if allocAmt.IsZero() {
```

Comment:
@kogisin Please check if this is a proper addition.

Reply:
Yes, I think this is a proper addition.